src/pages/round/[...number].tsx

```diff
@@ -65,81 +64,103 @@ export default function Round() {
+                          currency: "USD",
+                          maximumFractionDigits: 0,
+                        })}
                       </Text>
-                    </Box>
-                    <Divider h="2px" bg="#ED1200" my={3} />
-                    <VStack align="left">
-                      <HStack>
-                        <Text>{bribe.label}:</Text>
-                        <Text as="b">
-                          {bribe.rewardAmount.toLocaleString("en-US", {
-                            style: "currency",
-                            currency: "USD",
-                            maximumFractionDigits: 0,
-                          })}
-                        </Text>
-                      </HStack>
-                      <HStack>
-                        <Text>Vote Total:</Text>
-                        <Text as="b">
-                          {bribe.percent}% - {bribe.votes.toLocaleString("en-us")}
-                        </Text>
-                      </HStack>
-                      <HStack>
-                        <Text>$/1000 VP:</Text>
+                    </HStack>
+                    <HStack>
+                      <Text>Vote Total:</Text>
+                      <Text as="b">
+                        {bribe.percent}% - {bribe.votes.toLocaleString()}
+                      </Text>
+                    </HStack>
+                    <HStack>
+                      <Text>$/1000 VP:</Text>
+                      <Text as="b">
+                        {bribe.usdPer1000Vp.toLocaleString(undefined, {
+                          style: "currency",
+                          currency: "USD",
+                        })}
+                      </Text>
+                      <Popover placement="auto-start">
+                        <PopoverTrigger>
+                          <IconButton
+                            isRound
+                            height="0"
+                            minWidth="0"
+                            aria-label="Explain Item"
+                            icon={<Icon as={QuestionIcon} />}
+                          />
+                        </PopoverTrigger>
+                        <PopoverContent>
+                          <PopoverHeader fontWeight="semibold">$/1000 VP</PopoverHeader>
+                          <PopoverBody>
+                            Value in $ received for voting on this pool with 1k votes
+                          </PopoverBody>
+                        </PopoverContent>
+                      </Popover>
+                    </HStack>
+                    {accountConnected && voteStateActive && requestedRound === roundList.latest ? (
+                      <HStack marginTop="auto">
+                        <Text>Max single reward:</Text>
                         <Text as="b">
-                          {bribe.usdPer1000Vp.toLocaleString("en-US", {
+                          {((bribe.usdPer1000Vp * votingPower) / 1000).toLocaleString(undefined, {
```

Comment:
this will work, but we also have `bribe.usdPerVp` so you don't need to divide by 1000:
`(bribe.usdPerVp * votingPower).toLocaleString` ...

Reply:
good catch, i can't test this cause we are not in round ... but we can all take a look next vote